### ISSUE TITLE
How to use with enterprise github?

### ISSUE THREAD

**Issue by svacha** [tomgenoni/bitbar-ghissues#4]
Is there an optional `base_url` parameter that can be added to the config values [here](https://github.com/tomgenoni/bitbar-ghissues/blob/master/repo-issues.5m.js) so folks can connect enterprise github servers?

Maybe something similar to how PyGithub allows you to pass in the [`base_url` parameter](http://pygithub.github.io/PyGithub/v1/github.html?highlight=base_url).

**Comment by tomgenoni** [comment]
Ah, yeah, that should be doable. Could you give me an example of what a final enterprise url would look like?

**Comment by svacha** [comment]
Based on [this](https://developer.github.com/v3/enterprise/#endpoint-urls), enterprise APIs look like:

```
http(s)://hostname/api/v3/
```

Which makes me think that adding the following line to `repo-issues.5m.js`:

```
$HOSTNAME=""
```

and replacing the `host` line (4) of `lib/lib.js with something like:

```
host: $HOSTNAME || 'api.github.com',
```

That should do it. Then folks can set `$HOSTNAME=github.tinderforcats.com/api/v3` or whatever, and connect to their enterprise github instead.

**Comment by tomgenoni** [comment]
@svacha Can you pull the code from https://github.com/tomgenoni/bitbar-ghissues/tree/enterprise and let me know if that works? I made the change you suggested on line 4 but don't have a way to test it.

**Comment by tomgenoni** [comment]
Resolved by #5
